src/OnlineDnsLookup.php

```diff
@@ -43,27 +43,27 @@ public function getRecord(): ?string
     /**
      * Add a recursive reference that is included within this lookup's record.
      *
-     * @param self $reference
+     * @return $this
      */
-    public function addReference(self $reference): void
+    public function addReference(self $reference): self
     {
         $this->references[] = $reference;
+
+        return $this;
     }
 
     /**
      * Get all recursive references that are included in this lookup's record.
      *
-     * @return array<self>
+     * @return \SPFLib\OnlineDnsLookup[]
      */
     public function getReferences(): array
     {
-        return $this->references;
+        return $this->references[0];
```

Comment:
This change seems to cause an error. As `$this->references` is an array of `OnlineDnsLookup` instances, `$this->references[0]` will never return an array. `$this->references[0]` could also be undefined (if there are no references).

Reply:
Whoops, development leftover... thanks!